simulation/src/transportation/bird_mob.rs

```diff
@@ -0,0 +1,137 @@
+use crate::map::Map;
+use crate::map_dynamic::Itinerary;
+use crate::physics::{Collider, CollisionWorld, PhysicsGroup, PhysicsObject, Speed};
+use crate::utils::rand_provider::RandProvider;
+use crate::utils::resources::Resources;
+use crate::utils::time::GameTime;
+use crate::World;
+use common::rand::rand3;
+use egui_inspect::Inspect;
+use geom::{angle_lerpxy, Transform, Vec3};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Inspect)]
+pub struct BirdMob {
+    pub walking_speed: f32,
+    pub fly_anim: f32,
+}
+
+const PED_SIZE: f32 = 0.5;
+
+pub fn put_bird_in_coworld(coworld: &mut CollisionWorld, pos: Vec3) -> Collider {
+    Collider(coworld.insert(
+        pos.xy(),
+        PhysicsObject {
+            radius: PED_SIZE * 0.6,
+            group: PhysicsGroup::Unknown,
+            ..Default::default()
+        },
+    ))
+}
```

Comment:
Birds don't need to be queried by cars/pedestrians very efficiently so I don't think they need to be in the CollisionWorld